modules/nw-infw-operator-config-object.adoc

```diff
@@ -38,6 +38,11 @@ spec:
 One label used in `nodeSelector` must match a label on the nodes in order for the daemon set to start. For example, if the node labels `node-role.kubernetes.io/worker` and `node-type.kubernetes.io/vm` are applied to a node, then at least one label must be set using `nodeSelector` for the daemon set to start.
 ====
 
+|`spec.ebpfProgramManagerMode`
+|`boolean`
+|
+Specifies if the Node Ingress Firewall Operator uses the bpfman Operator to manage BPF programs. This capability is a Technology Preview feature. See "About the bfpman Operator" for more information.
```

Comment:
@jldohmann, not easily, because this is a module and we can't import the TP module into this module, although I could copy and paste the TP notice in its entirety here for clarity.